doc/stdlib/dates.rst

```diff
@@ -402,16 +520,14 @@ Adjuster Functions
 Periods
 ~~~~~~~
 
-.. function:: Year(v)
-              Month(v)
-              Week(v)
-              Day(v)
-              Hour(v)
-              Minute(v)
-              Second(v)
-              Millisecond(v)
-
-   .. Docstring generated from Julia source
+.. data:: Year(v)
```

Comment:
The original PR did switch it from function to data. Worth noting that all those functions do have docstrings attached.